scripts/build.js

```diff
@@ -3,6 +3,12 @@
 process.env.BABEL_ENV = 'production';
 process.env.NODE_ENV = 'production';
 
+
+const babelIE11 = require('./patch/babelIE11');
+const babelConfig = require('babel-preset-react-app');
+
+babelIE11(babelConfig);
```

Comment:
The `babel` config needs to be patched before the `webpack` config, otherwise we're too late and it won't take effect.